scripts/exportOpenApi.gradle

```diff
@@ -14,19 +14,18 @@ buildscript {
 }
 apply plugin: 'org.openapi.generator'
 
-def specFile = config.openApi.specFile
+
+// For a list of supported options see https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-gradle-plugin
 
 //tag::exportOpenApi[]
 task exportOpenApi (
         type: org.openapitools.generator.gradle.plugin.tasks.GenerateTask,
         group: 'docToolchain',
         description: 'exports OpenAPI specification to the asciidoc file') {
 
+    def specFile = findProperty("openApi.specFile")?:config.openApi.specFile
     if (!specFile) {
-        logger.info("\n---> OpenAPI specification file not found in Config.groovy (https://doctoolchain.github.io/docToolchain/#_exportopenapi)")
-        return
-    } else {
-        logger.info("Found OpenAPI specification in Config.groovy")
+        throw new GradleException("Missing property 'openApi.specFile': please provide the location of the OpenAPI specification file")
```

Comment:
These two lines break a lot of other tests. 
Can anyone explain me, why throwing an exception in `exportOpenApi` affects other tasks? As far as I can tell, none of those tasks depend on `exportOpenApi`.